src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -581,6 +771,23 @@ public static OptionData fromData(@Nonnull DataObject json)
         OptionType type = OptionType.fromKey(json.getInt("type"));
         OptionData option = new OptionData(type, name, description);
         option.setRequired(json.getBoolean("required"));
+        if (type == OptionType.INTEGER || type == OptionType.NUMBER)
+        {
+            if (!json.isNull("min_value"))
+            {
+                if (json.isType("min_value", DataType.INT))
+                    option.setMinValue(json.getLong("min_value"));
+                else if (json.isType("min_value", DataType.FLOAT))
+                    option.setMinValue(json.getDouble("min_value"));
+            }
+            if (!json.isNull("max_value"))
+            {
+                if (json.isType("max_value", DataType.INT))
+                    option.setMaxValue(json.getLong("max_value"));
+                else if (json.isType("max_value", DataType.FLOAT))
```

Comment:
not sure if the type of the field should be checked because, lets say, the user provides some invalid type as value so the call will be ignored without any information.

Reply:
Users putting in invalid json should be none of our concern, we should expect the json to have the correct types and format.